src/index.css

```diff
@@ -1,13 +1,37 @@
-body {
+
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+body{
+  font-family: 'Roboto Condensed', sans-serif;
+}
+a{
+  text-decoration: none;
+}
+li{
+  list-style: none;
 }
+
+:root{
```

Comment:
Add root to the top of the file

Reply:
i have updated it,thankyou for the suggesstion